### PR TITLE
Fill in readme.md

### PR DESCRIPTION
**Pull request by ghost**
Ladies and gentlemen readme.md file wishes to be reviewed.
connect to #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,53 @@
...
+
+## Motivation
+
+A Ragnarson Internship 2016 group project: from theory to practice.
+
+## Built with
+
+* Ruby on Rails (and gems: autoprefixer)
+* Bootstrap
+* postgresql
```
Comment (CallMeSandra) [inline README.md]: We also have rspec

Comment (Crsck) [inline README.md]: I'd remove the sole gem or add full list of them.

File: README.md [hunk 2]
```diff
@@ -1 +1,53 @@
...
+
+If you wish to take part in our project feel free to clone it (using terminal):
+
+```
+$ git clone https://github.com/Ragnarson-Internship2016/group-IV
+```
+
+## Deployment
+
+Run file filename.rb
```
Comment (CallMeSandra) [inline README.md]: what do you men by filename.rb?

File: README.md (hunk 1 above)
Comment (CallMeSandra) [inline README.md]: Ruby 2.3.1 
Rails 5.0.0

File: README.md (hunk 2 above)
Comment (ghost) [inline README.md]: well, were to "click" to run app?
need a hint

File: README.md (hunk 1 above)
Comment (ghost) [inline README.md]: Done!

File: README.md [hunk 3]
```diff
@@ -1 +1,59 @@
...
+
+A Ragnarson Internship 2016 group project: from theory to practice.
+
+## Built with
+
+* Ruby ```2.3.1``` on Rails ```5.0.0``` and gems:
+  * autoprefixer
+  * rspec
+* Bootstrap
+* postgreSQL
```
Comment (gotar) [inline README.md]: you can drop this part, no reason to mention about every gem, they will be added all the time

File: README.md [hunk 4]
```diff
@@ -1 +1,59 @@
...
+
+## Installation
+
+If you wish to take part in our project feel free to clone it (using terminal):
+
+```
+$ git clone https://github.com/Ragnarson-Internship2016/group-IV
+```
+
+## Deployment
```
Comment (gotar) [inline README.md]: more like Development, not deployment

File: README.md [hunk 5]
```diff
@@ -1 +1,59 @@
...
+
+Next, in browser address field type:
+``` localhost:3000 ```
+
+## Contributing
+
+All issues must follow steps:
+1. suggest an issue
+2. create a pull request with issue name
+3. each pull request will be reviewed by menthors
```
Comment (gotar) [inline README.md]: mentors

File: README.md [hunk 6]
```diff
@@ -1 +1,59 @@
...
+4. after menthors confirmation new issue will be merged with master.
```
Comment (gotar) [inline README.md]: mentors

**Comment by gotar** [comment]
you should get +1 from another team member before you move to check by me

File: README.md [hunk 7]
```diff
@@ -1 +1,51 @@
...
+If you wish to take part in our project feel free to clone it (using terminal):
+
+```
+$ git clone https://github.com/Ragnarson-Internship2016/group-IV
+```
+
+## Development
+
+In terminal, run command
+``` rails server ```
```
Comment (smt116) [inline README.md]: it should be:

```
$ rails server
```

or even bin/rails instead of rails as you have binstubs created

File: README.md [hunk 8]
```diff
@@ -1 +1,51 @@
...
+## Authors
+
+* [Sandra Koryszewska](https://github.com/sandrakoryszewska)
+* [Agata Salska](https://github.com/agatasalska)
+* [Aneta Woźniak](https://github.com/jeathli)
+* [Małgorzata Maksimowicz](https://github.com/gosiamaks)
+* [Asia Bukowska](https://github.com/abukowska)
+* [Basia Cierocka](https://github.com/basiacierocka)
+* [Agnieszka Kryjewska](https://github.com/almuredin)
+* [Konrad Ciarczynski](https://github.com/crsck)
```
Comment (smt116) [inline README.md]: I would sort the list by names or surnames :P

**Comment by smt116** [comment]
Is there any reason for having three commits instead of "Add README file"?

File: README.md (hunk 8 above)
Comment (ghost) [inline README.md]: I thought about it but can't decide

**Comment by ghost** [comment]
commits are my fault..

File: README.md (hunk 7 above)
Comment (ghost) [inline README.md]: I forgot about it, sorry

File: README.md [hunk 9]
```diff
@@ -11,42 +11,41 @@ Enjoy.
```
Comment (gosiamaks) [inline README.md]: I'd add name of the project (Luncher)and one-sentence description what this app is used for: searching recipes by ingredients; maybe we can also do what another group: put the link to User Stories?

**Comment by ghost** [comment]
maybe there should be this part too? 

## Functionality

**Guest path** (able to search recipes by chosen ingredient, _view_ the list of all recipes, _view_ random recipe ('Recipe of the day'), _view_ single recipe in details):
I am on the homepage:
1. Ingredient route:
- I choose the ingredient from the list
- I click submit button
- Then I am redirected to the _gallery of recipes_, which use this ingredient
- I click one recipe from the list
- Then I am redirected to the _details page_ with **_selected_** recipe
1. 'All recipes' route:
   - I click 'All recipes' link
   - Then I am redirected to the _gallery with all recipes_
   - I click one from the list
   - Then I am redirected to the _details page_ with **_selected_** recipe
2. 'Recipe of a day' route:
   - I click 'Recipe of a day'
   - Then I am redirected to the _details page_ with **_randomly selected_** recipe
3. Features:
   - Recipe can be flagged 'Like it'
   - One-step 'Go back' button on each _view_

**Comment by gosiamaks** [comment]
I like it! I don't know if this kind of info is directly in README (or just there's a link to it); If yes, I'd put it here :)

**Comment by ghost** [comment]
no problem to put it in separate file ;)

**Comment by agatasalska** [comment]
👍

File: README.md [hunk 10]
```diff
@@ -1 +1,52 @@
...
+
+## Installation
+
+If you wish to take part in our project feel free to clone it (using terminal):
+
+```
+$ git clone https://github.com/Ragnarson-Internship2016/group-IV
+```
+
+## Development
```
Comment (basiacierocka) [inline README.md:24]: add our link to heroku: https://group4.herokuapp.com/

Comment (gotar) [inline README.md:24]: +1

File: README.md [hunk 11]
```diff
@@ -1 +1,52 @@
...
+
+In terminal, run command
```
Comment (smt116) [inline README.md]: put a colon at the end

File: README.md [hunk 12]
```diff
@@ -1 +1,52 @@
...
+
+` $ rails server `
```
Comment (smt116) [inline README.md]: it should be ``` instead of ` if it is a code snippet block (` is for inline)

File: README.md [hunk 13]
```diff
@@ -1 +1,52 @@
...
+
+Next, in browser address field type:
+` localhost:3000 `
```
Comment (smt116) [inline README.md]: I would remove this part. `rails server` command will show what is a proper address

File: README.md [hunk 14]
```diff
@@ -1 +1,52 @@
...
+
+## Contributing
+
+All issues must follow steps:
+1. suggest an issue
```
Comment (smt116) [inline README.md]: "All issues must follow steps: ... suggest an issue"

so issue should suggest an issue? :P

**Comment by ghost** [comment]
check please

**Comment by gotar** [comment]
You can merge. No reason to spend more time doing this.

**Pull request merged**